core/src/poh_recorder.rs

```diff
@@ -136,6 +136,33 @@ impl PohRecorder {
         self.ticks_per_slot
     }
 
+    fn ignore_grace_ticks(&self, slot: Slot) -> bool {
+        !(slot.saturating_sub(NUM_CONSECUTIVE_LEADER_SLOTS)..slot).any(|i| {
```

Comment:
wrapping a `!` around the entire function body is a little hard on my brain.

wdyt about removing the `!` and renaming the function from `ignore_grace_ticks()` to `received_previous_leader_data()`.

Reply:
Yes, that much cleaner. I'll update the code. Thanks!!